cmd/buildah/bud.go

```diff
@@ -301,6 +301,11 @@ func budCmd(c *cobra.Command, inputArgs []string, iopts budOptions) error {
 
 	capabilities := defaultContainerConfig.Capabilities("", iopts.CapAdd, iopts.CapDrop)
 
+	arch, os, err := parse.PlatformFromOptions(c)
```

Comment:
These values must be flipped. It's `os, arch`.

Reply:
Hah, that's a great catch :) thanks